internal/http/web/application.go

```diff
@@ -30,6 +30,13 @@ func (h *Handler) getApply() http.HandlerFunc {
 				h.Error(w, r, err, "")
 				return
 			}
+		} else {
+			// Show flash that app has been saved
+			session.AddFlash(flash.Flash{
+				Type:    flash.Success,
+				Message: "You're application has been submitted! feel free to update it here until applications close.",
```

Comment:
Literally unreadable
```suggestion
				Message: "Your application has been submitted! Feel free to update it here until applications close.",
```